request/graphql/parser/commit.go

```diff
@@ -37,9 +37,9 @@ func parseCommitSelect(schema gql.Schema, parent *gql.Object, field *ast.Field)
 		} else if prop == request.Cid {
 			raw := argument.Value.(*ast.StringValue)
 			commit.Cid = immutable.Some(raw.Value)
-		} else if prop == request.FieldName {
+		} else if prop == request.FieldIDName {
```

Comment:
thought(out-of-scope): I'm noticing that the `prop` is being compared to field names and we have the proper constants to make it a more visually descriptive conditional.

```Go
LinksFieldName           = "links"
HeightFieldName          = "height"
CidFieldName             = "cid"
DockeyFieldName          = "dockey"
CollectionIDFieldName    = "collectionID"
SchemaVersionIDFieldName = "schemaVersionId"
FieldNameFieldName       = "fieldName"
FieldIDFieldName         = "fieldId"
DeltaFieldName           = "delta"
```